service/src/main/java/com/fecklessweasel/service/UserSessionUtil.java

```diff
@@ -0,0 +1,173 @@
+package com.fecklessweasel.service;
+
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.SQLException;
+import javax.servlet.http.Cookie;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.Hashtable;
+
+import com.fecklessweasel.service.datatier.SQLSource;
+import com.fecklessweasel.service.datatier.SQLInteractionInterface;
+import com.fecklessweasel.service.objectmodel.CodeContract;
+import com.fecklessweasel.service.objectmodel.ServiceException;
+import com.fecklessweasel.service.objectmodel.ServiceStatus;
+import com.fecklessweasel.service.objectmodel.UserSession;
+
+/**
+ * Utility class for creating and deleting UserSessions. Encapsulates cookie
+ * operations required for log in and logout.
+ * @author Christian Gunderman
+ */
+public final class UserSessionUtil {
+    /** The client side cookie for storing user authentication token. */
+    private static final String AUTH_COOKIE = "FecklessWeaselAuthToken";
```

Comment:
While I agree with you, I don't think this constant should be universal. This class was created to encapsulate the session cookie activities for maintainability. By keeping the constant here we are discouraging modification or reference to the cookie elsewhere.